content/sensu-go/6.8/catalog/catalog-reference.md

```diff
@@ -263,12 +263,14 @@ The catalog-api tool is written in Go.
 
 ### catalog-api subcommands
 
-The catalog-api tool provides the following subcommands:
+The catalog-api tool provides the following subcommands.
+Replace <REPO_NAME> with the name of your local repository of integrations:
 
 {{< code text >}}
-catalog-api catalog --help
+catalog-api <REPO_NAME> --help
+
 USAGE
-  catalog-api catalog [flags] <subcommand> [flags]
+  catalog-api <REPO_NAME> [flags] <subcommand> [flags]
```

Comment:
I think the usage was previously correct here. There aren't any arguments for a repo name but there is a flag for repo dir (`--repo-dir path/to/catalog/repo`).